cfy_manager/components/restservice/restservice.py

```diff
@@ -268,6 +268,8 @@ def _configure_db():
     if config[CLEAN_DB]:
         db.prepare_db()
         db.populate_db()
+    else:
+        db.create_amqp_resources()
```

Comment:
if we are to have this separate script, i think we should be always running it, and we should make the `create-tables-blabla.py` script not do that part
i mean, just look at this function, which is called from the `create-tables.py`: https://github.com/cloudify-cosmo/cloudify-manager/blob/43b8a667be1ea078b24554f22f19b45369c6761e/rest-service/manager_rest/storage/storage_utils.py#L53
it's only using `amqp_manager` for the call that is done in the amqp script
so let's get rid of amqp altogether from the 1st script, and call both scripts when creating a clean db

Reply:
In general, you're right of course. The thing was that this would've required changes in cloudify-manager, which I wanted to avoid. Do you think it's worth it?

Reply:
i do think it's worth it, but yeah, maybe let's do this one first, and separate later